sgx/urts/src/panic.rs

```diff
@@ -15,11 +15,17 @@ pub unsafe extern "C" fn report_panic_message(msg: *const u8, msg_len: usize) {
 
 fn report_panic_message_impl(panic_msg_bytes: &[u8]) {
     match str::from_utf8(panic_msg_bytes) {
-        Ok(v) => global_log::crit!("Enclave panic:\n{}\n", v),
-        Err(e) => global_log::crit!(
-            "Enclave panic message contained invalid utf8:\n{}\n{:?}",
-            e,
-            panic_msg_bytes
-        ),
+        Ok(v) => {
+            eprintln!("Enclave panic:\n{}\n", v);
+            global_log::crit!("Enclave panic:\n{}\n", v)
+        },
+        Err(e) => {
+            eprintln!("Enclave panic message contained invalid utf8:\n{}\n{:?}", e, panic_msg_bytes);
+            global_log::crit!(
+                "Enclave panic message contained invalid utf8:\n{}\n{:?}",
+                e,
+                panic_msg_bytes
+            )
+        },
```

Comment:
Eran felt strongly that this should be logged using slog, that's why we closed the first iteration some years ago.
I think its okay to log to both, I don't see any downside, and since they go to STDERR and STDOUT it's not going to confuse any log parser.

Reply:
I can try to add a code comment i guess about this, to help the next guy who looks at this